### DISCUSSION TITLE
Downvote button

### DISCUSSION THREAD

**Discussion by kennethfreitas** [dailydotdev/daily#460] [Feature Requests]
As we see the trouble that YT created by removing the dislike button the dailydotdev suffers from the same problem. As blogging gets something "necessary" to devs, open websites like dev.to or medium get a lot of bad content and misinformation. With a downvote button, we can easily filter and get data from bad authors to remove priority from them.

**Comment by nimrodkra** [comment]
Hi @kennethfreitas, thank you so much for the thoughtful feature request! This is not the first time we received it. However, the way you described the need behind the downvoting feature sparked a discussion within our team! We can't make any promises at this stage, but we are definitely going to seriously consider a way to implement such an option. 

I'll keep you posted here once we have updates to share.

**Reply by ColaIan** [reply]
Hi it has been 10 months, just wondering if your team has concluded, or when will your team be implementing the feature as I see the feature request has an "accepted" label?

**Reply by idoshamun** [reply]
The downvote feature is on our roadmap and already has designs ready. Unfortunately, we have some technical constraints to solve before implementing it, and there are higher prioritized features. I hope we can deliver it in the upcoming months. With that said, feel free to use the report/block option when it makes sense

**Reply by ColaIan** [reply]
Thanks for the info! I guess the design is probably just gonna look like an upside down upvote button haha; anyway I was just gonna downvote posts that are too clickbaity and doesn't suit my preference, so in this case I'm probably not gonna use (abuse) the report/block buttons and will just wait for the feature, thanks!

**Reply by idoshamun** [reply]
Hehe! Actually, we're going to do something a bit more advanced. See below (design might change)
<img width="710" alt="image" src="https://user-images.githubusercontent.com/1993245/205484584-03077ffa-309a-452c-ae3c-a1cf95b836d2.png">

**Reply by killshot13** [reply]
Hey👋

I was going to submit a feature request for downvoting articles, but I decided to do my due diligence first to see if anyone had already requested something similar; then, I discovered this thread.

---

I love the concept shown in the image on the right; it ties right into what has always bugged me about the box with the sharing options that read, "Would you recommend this post?"

In most cases, my answer is "Yes!" and I happily copy the link or retweet the article. But occasionally, a post or tutorial might suck. Once in a while, I read a blog post, and when I am done, I feel that I have lost brain cells and been robbed of five minutes of my life that I'll never get back. 

And when now I see this question, my answer is a resounding "NO!" 

![image](https://github.com/dailydotdev/daily/assets/47695475/779c74c6-b802-40b2-b53f-416c1ddc2cb2)

But unfortunately, there is no way for me to express that sentiment. I want to warn others not to waste their time reading the article by downvoting it. Or, at the very least, even if a numerical count of user downvotes weren't publicly visible, they could be used behind the scenes to influence the weighted algorithm by reducing the likelihood of a post appearing in the trending or top-rated sections. 

Just like positive responses help an article get more views, downvotes could be used to, once a certain threshold is exceeded, reduce that post's visibility across the board for all users rather than just on a per-account basis.

---

FYI, in case my suggestion is too far off-topic from what you are working on here, please let me know, and I will gladly move it to a separate discussion. Not trying to resurrect/hijack a thread that isn't mine.🙃

**Reply by ColaIan** [reply]
I agree with you, it makes no sense not having a no button for a yes no question lmao

**Reply by idoshamun** [reply]
Agree with both of you for the clarity of doubt. Really hope we can deliver it soon.
Our team is small and heavily focused on squads right now.

**Reply by dcraig327** [reply]
> Hehe! Actually, we're going to do something a bit more advanced. See below (design might change) <img alt="image" width="710" src="https://user-images.githubusercontent.com/1993245/205484584-03077ffa-309a-452c-ae3c-a1cf95b836d2.png">

This is brilliant! 

If you were to ever add the opposite feature and allow users to say "this article is about <tag>", consider bias. For example if during registration you were to ask "Do you work for a media company or are you a dev?" as even though this is called daily.dev - everyone is biased are you may have the press doing the opposite of what devs feel is right. So asking that question would sure provide some interesting stats for your team to review.

Regarding bias. I highlighted another one built into the platform when selecting tags the bottom paragraph of https://github.com/dailydotdev/daily/discussions/905

**Comment by nimrodkra** [comment]
**UPDATE:** Downvotes are now live 🎉 [Read the changelog](https://app.daily.dev/posts/QK301jNUY)

![Downvote articles (3)](https://github.com/dailydotdev/daily/assets/41470823/193a2ae8-965a-44f4-bb27-3aa8b97e161b)
